cypress/support/commands.js

```diff
@@ -4,7 +4,7 @@
 // https://on.cypress.io/custom-commands
 // ***********************************************
 
-import { JsonRpcProvider } from '@ethersproject/providers'
+import { JsonRpcProvider } from 'ethers_providers_alfetopito'
```

Comment:
Cool, let's add an issue also so we don't forget to bring this back to etheres.
Awesome job patching them